### DISCUSSION TITLE
Doubt regarding the project

### DISCUSSION THREAD

**Discussion by jaiakash** [DiveHQ/backend-internship-task#21] [Q&A] [answered]
I have few questions.
- It is asked to create REST API, so just the backend is enough (lets say using OpenAPI and swagger)
or do we need to create the frontend as well for displaying data eg showing user calorie, expected number of calories per day, etc.
- The requirement given in the task is to use any Python framework. If the answer to previous ques is to create frontend as well, then can we use non-python framework like React or Flutter for frontend.

**Comment by prayashm** [comment, marked as answer]
> or do we need to create the frontend as well for displaying data eg showing user calorie, expected number of calories per day, etc.

Nope, no frontend. Only the API. Postman collection is a plus.
